server/Common/Utils/JwtUtil.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+
+namespace Common.Utils
+{
+    public class JwtUtil
+    {
+        /// <summary>
+        /// Extracts user id from jwt token
+        /// </summary>
+        /// <param name="token">JWT token generated by IdentityService</param>
+        /// <returns>User identifier</returns>
+        public static Guid GetUserIdFromToken(string token)
+        {
+            token = removeBearer(token);
+
+            var readToken = new JwtSecurityTokenHandler().ReadJwtToken(token);
+            var userId = ((List<Claim>) readToken.Claims).Find(c => c.Type.Equals("sub"));
+            return Guid.Parse(userId.Value);
+        }
+
+        private static string removeBearer(string token)
```

Comment:
Do we have a convention of naming private methods with a non-capital letter? I'm rather accustomed to what I think is a standard of naming methods with first capital letter no matter their access modifier

Reply:
Nah, just didn't notice :p I think that I have some habits from other languages :)